active-rfcs/0000-new-scoped-slots-syntax.md

```diff
@@ -259,10 +333,90 @@ The change is fully backwards compatible, so we can roll it out in a minor relea
 
 `slot-scope` is going to be soft-deprecated: it will be marked deprecated in the docs, and we would encourage everyone to use / switch to the new syntax, but we won't bug you with deprecation messages just yet because we know it's not a top priority for everyone to always migrate to the newest stuff.
 
-In 3.0 we do plan to eventually remove `slot-scope`, and only support `slot-props` and its shorthand. We will start emitting deprecation messages for `slot-scope` usage in the next 2.x minor release to ease the migration to 3.0.
+In 3.0 we do plan to eventually remove `slot-scope`, and only support the new syntax. We will start emitting deprecation messages for `slot-scope` usage in the next 2.x minor release to ease the migration to 3.0.
 
 Since this is a pretty well defined syntax change, we can potentially provide a migration tool that can automatically convert your templates to the new syntax.
 
 # Unresolved questions
 
-- Settle on a good shorthand symbol
+### Shorthand Syntax
+
+In a previous draft of this proposal, we proposed a shorthand for the new syntax:
+
+``` html
+<foo ()="{ msg }">
+  {{ msg }}
+</foo>
+
+<foo>
+  <template (header)="{ msg }">
+    Message from header: {{ msg }}
+  </template>
+
+  <template (footer)>
+    A static footer
+  </template>
+</foo>
+```
+
+Here the `v-slot:arg` is translated into `(arg)`.
+
+However, there were very mixed opinions on which symbol should be used for the shorthand, or whether a shorthand is needed at all. Two most favored alternative symbols for the shorthand are listed below:
+
+### `#`
+
+``` html
+<foo #="{ msg }">
+  {{ msg }}
+</foo>
+
+<foo>
+  <template #header="{ msg }">
+    Message from header: {{ msg }}
+  </template>
+
+  <template #footer>
+    A static footer
+  </template>
+</foo>
+```
+
+### `&`
+
+``` html
+<foo &="{ msg }">
+  {{ msg }}
+</foo>
+
+<foo>
+  <template &header="{ msg }">
+    Message from header: {{ msg }}
+  </template>
+
+  <template &footer>
+    A static footer
+  </template>
+</foo>
+```
+
+Personally, I think a shorthand is a beneficial addition, as it makes the slot designation jump out from normal props and directives and makes the template easier to scan.
+
+After the initial round of feedback I agree that `()` as the shorthand can be potentially confusing to beginners, and am now leaning towards `#`. Some examples of the usage in a real-world library that relies on scoped slots ([vue-promised](https://github.com/posva/vue-promised)):
+
+``` html
+<Promised :promise="usersPromise">
+  <template #pending>
+    <p>Loading...</p>
+  </template>
+
+  <template #="users">
+    <ul slot-scope="users">
```

Comment:
the `slot-scope="users"` on the `ul` is no longer needed, correct?

Reply:
thanks, fixed